packages/datadog_flutter_plugin/lib/src/rum/rum_user_action_detector.dart

```diff
@@ -195,12 +195,17 @@ class _RumUserActionDetectorState extends State<RumUserActionDetector> {
 
   _RumTreeAnnotation? _findElementInnerText(Element element, bool allowText) {
     String? elementDescription;
+    Map<String, Object?>? attributes;
 
     void visitor(Element element) {
       bool stopVisits = false;
 
       var widget = element.widget;
-      if (allowText && widget is Text) {
+      if (allowText && widget is RumUserActionAnnotation) {
```

Comment:
Not sure whether the `allowText` is needed here, maybe we should consider that a `RumUserActionAnnotation` widget should always "win" the search text as we consider it the most descriptive widget no matter what ?

Reply:
No, the `allowText` shouldn't be needed. It's there to prevent accidentally pulling text from something that might have sensitive information, but since `RumUserActionAnnotation` isn't an automatic capture, users should be able to precent them on their own.